src/js/bootstrap-slider.js

```diff
@@ -973,14 +983,23 @@ const windowIsDefined = (typeof window === "object");
 			- Any method that is not part of the public interface.
 			- Place it underneath this comment block and write its signature like so:
 
-			  					_fnName : function() {...}
+				_fnName : function() {...}
 
 			********************************/
 			_removeSliderEventHandlers: function() {
 				// Remove keydown event listeners
 				this.handle1.removeEventListener("keydown", this.handle1Keydown, false);
 				this.handle2.removeEventListener("keydown", this.handle2Keydown, false);
 
+				//rmove the listeners from the ticks if they had their own listeners
+				if(this.options.ticks_tooltip){
+					var ticks = document.getElementsByClassName('slider-tick-container')[0].children;
+					for(var i = 0; i < ticks.length; i++ ){
+						ticks[i].removeEventListener('mouseenter', this.createTickMouseOverListener, false);
```

Comment:
So this is not actually removing the `mouseenter` and `mouseleave` event listeners from the ticks.

Within the `this._createTickMouseOverListener` function, you are binding event listeners for the events. However, when `removeEventListener` is called, you need to pass in the specific function reference for the actual listener itself.

For example:

```js
// Create a function reference
var myEventListener = function() { console.log("yolo"); };

// Create an event listener by binding a DOM event to the specific function reference
myElementReference.addEventListener("mouseenter", myEventListener);

// This line properly removes the listener because myEventListener refers to the same memory reference that was originally bound in the line above
myElementReference. removeEventListener("mouseenter", myEventListener, false);
```